templates/todo/projects/csharp-sql/README.md

```diff
@@ -14,19 +14,9 @@ Let's jump in and get the ToDo app up and running in Azure. When you are finishe
 
 The following prerequisites are required to use this application. Please ensure that you have them all installed locally.
 
-- [Azure Developer CLI](https://aka.ms/azure-dev/install)
-  - Windows:
-    ```powershell
-    powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"
-    ```
-  - Linux/MacOS:
-    ```
-    curl -fsSL https://aka.ms/install-azd.sh | bash
-    ```
-- [Azure CLI (2.38.0+)](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [Azure Developer CLI](https://aka.ms/azd-install)
```

Comment:
I would like to follow https://aka.ms/azd/install pattern if possible

Reply:
I am reusing an existing aka.ms used in Savannah's blog announcement.